test/record.cooltt

```diff
@@ -6,11 +6,13 @@ def basic : type :=
     (foo.x : nat)
     (bar : nat → nat)
 
-def basic.inhabit : basic := struct (foo.x : 1) (bar : x => x)
+def basic.inhabit : basic := struct (foo.x : 1) (bar : x => suc x)
```

Comment:
Hmm... Should we parse the standalone `suc` as the successor function?

Reply:
Perhaps!